src/NServiceBus.AcceptanceTests/RunDescriptorExtensions.cs

```diff
@@ -1,20 +0,0 @@
-﻿namespace NServiceBus.AcceptanceTests
-{
-    using System;
-    using AcceptanceTesting.Support;
-    using ScenarioDescriptors;
-
-    public static class RunDescriptorExtensions
-    {
-        public static Type GetTransportType(this RunDescriptor runDescriptor)
-        {
-            Type transportType;
-            if (!runDescriptor.Settings.TryGet("Transport", out transportType))
```

Comment:
Where does this "Transport" type come from? Is it specific to the testing framework? If so, do we need to clean up the code that stores it as well, or is it still used in other places?